.ci/test.sh

```diff
@@ -51,11 +51,20 @@ if [[ $TRAVIS == "true" ]] && [[ $TASK == "check-docs" ]]; then
 fi
 
 if [[ $TASK == "lint" ]]; then
-    conda install -q -y -n $CONDA_ENV pycodestyle pydocstyle
+    conda install -q -y -n $CONDA_ENV \
+        pycodestyle \
+        pydocstyle \
+        r-argparse \
```

Comment:
Is it light-weighted in terms of dependencies package? I mean, can avoid installing this library for the linting check with the aim to speed them up?

Reply:
You know, since I'm only passing a single argument I can probably just use `commandArgs()` in the script and avoid this altogether. Let me try that. One less dependency to worry about, you are right.